src/components/Dashboard/Header/Authentication/Logout.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import Redirect from 'react-router-dom/es/Redirect';
```

Comment:
I guarantee, importing like that will definitely crash testing attempts and keep you awake for hours before you realise where the problem is.

Reply:
What is wrong with this import? Please, provide how it should be fixed.

Reply:
Jest cannot find paths specified this way leaving only vague explanations. What you should have done is this:
`import { Redirect } from 'react-router-dom'`